documentation/source/install.rst

```diff
@@ -34,8 +34,11 @@ The installation instructions vary depending on your operating system:
 
    .. group-tab:: Windows
 
-      `Conda <https://conda.io/>`_ is an open-source package and environment management system that we recommend for Windows.
+      We recommend users who are running Windows, and who are more comfortable with a Unix shell or who have legacy Makefile-based projects,
+      to use either Windows Subsystem for Linux (WSL) or a Virtual Machine, such as Oracle VirtualBox.
```

Comment:
```suggestion
      We recommend users who are running Windows and who are more comfortable with a Unix shell or who have legacy Makefile-based projects to use either Windows Subsystem for Linux (WSL) or a Virtual Machine such as Oracle VirtualBox.
```
Better grammar I think? @cmarqu

Reply:
Uh, not sure really... based on feeling I might even add a comma before the "or who have".